aikau/src/main/resources/alfresco/services/_PreferenceServiceTopicMixin.js

```diff
@@ -48,7 +49,7 @@ define(["dojo/_base/declare",
        * @type {string}
        * @default
        */
-      addFavouriteDocumentTopic: "ALF_PREFERENCE_ADD_DOCUMENT_FAVOURITE",
+      addFavouriteDocumentTopic: topics.ADD_FAVOURITE_NODE,
```

Comment:
It'd be good to define these member variables (and the other one below) as events, and to list the functions using them as listening/firing them, so that we can add a listens (or fires) tag to the member variables and link them back to the topics file in the JSDoc.